client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -141,7 +141,7 @@ const videoPressTvFlows: Record< string, () => Promise< { default: Flow } > > =
 	  }
 	: {};
 
-const aiAsseblerFlows: Record< string, () => Promise< { default: Flow } > > = config.isEnabled(
```

Comment:
[nit] I think we don't need the feature flag to control the flow as the flow is loaded async and there is no entry point to the ai-assembler 🙂

Reply:
That's a good point. WDYT about redirecting users to the assembler-first flow if it's not on horizon. Or maybe it's not needed since there's no entry point 🤔

Reply:
I think it's not needed 😆